src/js/view/utils/skin.js

```diff
@@ -156,30 +156,30 @@ export function handleColorOverrides(playerId, skin) {
         }
 
         addStyle([
-            '.jw-controlbar'
+            '.jw-controlbar',
+            '.jw-settings-topbar'
         ], 'background', config.background);
     }
 
     function styleTimeslider(config) {
 
         addStyle([
             '.jw-progress',
-            '.jw-buffer',
             '.jw-knob'
-        ], 'background', 'none ' + config.progress);
+        ], 'background-color', config.progress);
```

Comment:
By only setting `background-color`, this would not remove other styles applied to the background. The goal here is to reset the background and add the customization.

Reply:
It's unlikely that other background properties would be applied to the time slider. The customization options are intended for setting background color in general.